MalwareAnalyzer/views.py

```diff
@@ -85,6 +85,14 @@ def update_malware_db():
     """Check for update in malware DB"""
     try:
         url = "https://www.malwaredomainlist.com/mdlcsv.php"
+        if settings.UPSTREAM_PROXY_ENABLED:
```

Comment:
`settings.UPSTREAM_PROXY_ENABLED` is not required. Remove it from `settings.py` replace the logic with
`if settings.UPSTREAM_PROXY_IP and settings.UPSTREAM_PROXY_PORT`

Reply:
@ajinabraham  I put a UPSTREAM_PROXY_ENABLED to have the same structure as VirusTotal, I was also thinking that it coudl be helpful to be able to disabel proxy without wipping informations like  ip address , port , username and password but if you prefer I will change the code and remove usage of UPSTREAM_PROXY_ENABLED

Reply:
@superpoussin22 That make sense. I agree with you. Let's have that settings option.